config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/netobserv/network-observability-operator
-  newTag: main
+  newName: quay.io/jotak/network-observability-operator
+  newTag: v0.0.1-7
```

Comment:
I think this change was not supposed to be commited.

Reply:
yes it was committed by mistake, same for the port changes in CR below, I reverted these changes